articles/api/management/v2/user-search.md

```diff
@@ -66,17 +71,16 @@ The user data for `jane` would match, but `jane-doe` would not.
 
 The fields which support `raw` subfield queries are as follow:
 
-* `identities.connection﻿⁠⁠⁠⁠`
-* ﻿⁠⁠⁠⁠`identities.provider﻿⁠⁠⁠⁠`
-* ﻿⁠⁠⁠⁠`identities.user_id ﻿⁠⁠⁠⁠`
-* ﻿⁠⁠⁠⁠`email﻿⁠⁠⁠⁠`
-* ﻿⁠⁠⁠⁠`phone_number﻿⁠⁠⁠⁠`
-* ﻿⁠⁠⁠⁠`family_name﻿⁠⁠⁠⁠`
-* ﻿⁠⁠⁠⁠`given_name﻿⁠⁠⁠⁠`
-* ﻿⁠⁠⁠⁠`username﻿⁠⁠⁠⁠`
-* ﻿⁠⁠⁠⁠`name﻿⁠⁠⁠⁠`
-* ﻿⁠⁠⁠⁠`nickname﻿⁠⁠⁠⁠`
-
+* `identities.connection⁠⁠⁠⁠`
+* ⁠⁠⁠⁠`identities.provider⁠⁠⁠⁠`
+* ⁠⁠⁠⁠`identities.user_id⁠⁠⁠⁠`
+* ⁠⁠⁠⁠`email⁠`
+* ⁠⁠⁠⁠`phone_number⁠⁠`
+* ⁠⁠⁠⁠`family_name⁠⁠⁠⁠`
+* ⁠⁠⁠⁠`given_name⁠⁠⁠⁠`
+* ⁠⁠⁠⁠`username⁠⁠⁠⁠`
+* ⁠⁠⁠⁠`name⁠⁠`
+* ⁠⁠⁠⁠`nickname`
 
 ## Example Queries
 
```

Comment:
Alternatively, I would convert all these sub-paragraphs to a matrix, it would look better I think:

| Use Case | Query |
|:-----|:----|
| Search across all fields for "john" | `john` |
| Search for all users whose name _contains_ "john" | `name:"john"` |
| Search all users whose name _is_ exactly "john" | `name.raw:"john"` |